R/utils.R

```diff
@@ -21,3 +21,42 @@ epidist_version_stanvar <- function() {
   comment <- paste0("// code chunks used from epidist ", version, "\n")
   brms::stanvar(scode = comment, block = "functions")
 }
+
+#' Replace a brms prior only if it exists
+#'
+#' @param old_prior One or more prior distributions in the class `brmsprior`
+#' @param new_prior One prior distribution in the class `brmsprior`
+#' @family utils
+#' @importFrom cli cli_inform
+#' @autoglobal
+#' @export
+replace_prior <- function(old_prior, new_prior) {
+  if (is.null(new_prior)) {
+    return(old_prior)
+  }
+  cols <- c("class", "coef", "group", "resp", "dpar", "nlpar", "lb", "ub")
+  prior <- dplyr::full_join(
```

Comment:
I'm using `dplyr` because I don't know `data.table` well. Maybe an issue to replace this with `data.table` eventually

Reply:
https://github.com/epinowcast/epidist/issues/246